tools/debug-ui/src/BlocksView.tsx

```diff
@@ -0,0 +1,188 @@
+                                    <td>{block.in_progress_ms} ms</td>
+                                    <td>{printTimeInMs(block.orphaned_ms)}</td>
+                                    <td>{printTimeInMs(block.missing_chunks_ms)}</td>
+                                    {block.chunks_info!.map((chunk) => {
+                                        if (chunk) {
+                                            return (
+                                                <td key={chunk.shard_id}>
+                                                    <strong>
+                                                        {' '}
+                                                        {chunk.status}{' '}
+                                                        {getChunkStatusSymbol(chunk.status)}{' '}
+                                                    </strong>
+                                                    {chunk.completed_timestamp ? (
+                                                        <>
+                                                            {' '}
+                                                            <br /> Completed @ BR
+                                                            {printDuration(
+                                                                block.received_timestamp,
+                                                                chunk.completed_timestamp
+                                                            )}{' '}
+                                                        </>
+                                                    ) : (
+                                                        <></>
+                                                    )}
+                                                    {chunk.requested_timestamp ? (
+                                                        <>
+                                                            {' '}
+                                                            <br /> Requested @ BR
+                                                            {printDuration(
+                                                                block.received_timestamp,
+                                                                chunk.requested_timestamp
+                                                            )}{' '}
+                                                        </>
+                                                    ) : (
+                                                        <></>
+                                                    )}
+                                                    {chunk.request_duration ? (
+                                                        <>
+                                                            {' '}
+                                                            <br /> Duration {
+                                                                chunk.request_duration
```

Comment:
printTimeInMs? (or just delete that function)

Reply:
Cool, i copied paste it from here, https://github.com/near/nearcore/blob/f4e882aa6a000eec5f7e2166a858ee12b9cc8267/chain/jsonrpc/res/chain_n_chunk_info.html#L104

but updated to use the printTimeInMs instead